### PR TITLE
feat: UNEPLogo on footer

### PR DESCRIPTION
**Pull request by sunilsabatp**
Fixes #

Changes in this pull request:
- added the UNEP logo at the footer

@

### CODE REVIEW

File: src/Layout/Footer.tsx [hunk 1]
```diff
@@ -105,6 +106,9 @@ function Footer({}: Props): ReactElement {
               height={34}
             />
           </a>
+          <div>
+            <UNEPLogo/>
+          </div>
```
Comment (tejassonar) [inline src/Layout/Footer.tsx:111]: Please put the logo in an anchor tag with target blank and href="https://www.unep.org/". Refer web-app code for more details.

**Comment by delightedMatt** [comment]
Please add the text between the logos

**Comment by norbertschuler** [comment]
> Please add the text between the logos

Where is the spec for these texts?

**Review by tejassonar — APPROVED** [review]

**Comment by tejassonar** [comment]
LGFM

File: src/Layout/Footer.tsx [hunk 2]
```diff
@@ -44,73 +45,96 @@ function Footer({}: Props): ReactElement {
...
+                >
+                  <UNEPLogo />
+                </a>
+              </div>
+            </div>
+          </div>
+          <div className="planet-support-text">
+            <p style={{ fontStyle: "italic" }}>
+              Plant-for-the-Planet Supports the United Nations Environment
+              Program{" "}
```
Comment (sagararyal) [inline src/Layout/Footer.tsx:127]: This needs to be translated.

**Pull request merged**